src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -101,12 +102,13 @@
      (let [{:keys [library-schema questions enumerate]} props]
        [(if enumerate :ol :div) {}
         (map
-          (fn [{:keys [property required inputHint renderHint]}]
+          (fn [{:keys [property required inputHint renderHint editable]}]
             (let [property-kwd (keyword property)
-                  {:keys [title type enum minimum consentCode]} (get-in library-schema [:properties property-kwd])
+                  {:keys [title type typeahead enum minimum consentCode]} (get-in library-schema [:properties property-kwd])
                   error? (contains? (:invalid-properties @state) property-kwd)
                   colorize (fn [style] (merge style (when error? {:borderColor (:exception-state style/colors)})))
-                  update-property #(swap! state update :attributes assoc property-kwd (.. % -target -value))]
+                  update-property #(swap! state update :attributes assoc property-kwd (.. % -target -value))
+                  disabled? (false? editable)]
```

Comment:
So ... I'm confused about what's happening here - I think your rebase is off. This might stomp on the LMSVN commits but I can't tell what git will do in the end. If I compare this branch's version against origin/dev, these lines are not changed.